Library/functions.php

```diff
@@ -206,7 +207,7 @@ function is_windows()
  */
 function is_macos()
 {
-    return 'DARWIN' === strtoupper(substr(PHP_OS, 0, 6));
+    return 0 === stripos(PHP_OS, 'DARWIN');
 }
 
```

Comment:
`strtolower()` is extraneous with a case-insensitive check.

Reply:
@danhunsaker  `stripos`  the same as `strtoupper` (`strtolower`) + `substr`, isn't

Reply:
This stupid thing put the comment in the wrong part of the code. This specific line is perfectly fine (though I'm of a mind to use `-1 <=` instead of `0 ===`).